Development/NutriMon/src/main/resources/config.properties

```diff
@@ -1,4 +1,4 @@
-dbpassword = 11111111
+dbpassword = root
```

Comment:
Undo this change (different local password)

Reply:
This is fine though